src/js/mdRainbow.js

```diff
@@ -502,6 +504,8 @@ angular.module('mdRainbow', [])
 
 				// Defaults
 				// Everything is enabled by default.
+				$scope.okText = $scope.okText === undefined ? 'Select' : $scope.okText;
+				$scope.cancelText = $scope.cancelText === undefined ? 'Cancel' : $scope.cancelText;
```

Comment:
Could you turn these default values for `okText` and `cancelText` ('Select' and 'Cancel') into variables on the controller? Something like `$scope.defaultOkText` and `$scope.defaultCancelText` would be fine.